.gitignore

```diff
@@ -0,0 +1,2 @@
+/storage/
+foo.*
```

Comment:
More thinking out loud; instead of ignoring `foo.*`, let's write code that writes `foo.*` to the correct `/storage/output` location but comment that code to indicate that it's a workaround?